conboarding.md

```diff
@@ -191,3 +191,21 @@ How much does it cost to run things?
 
 4. ReproNim: request iam from David for AWS Access
 5. DANDI: request credentials for DANDI from Satra
+
+## Collaboration on Pull Requests
+
+The CON team works on many projects and is often spread thin, so its important to be efficient with pull requests and review.
+
+### Creating PRs
+- Keep PRs as small and focused as possible - this makes them easier for others to review and has a lower risk of introducing unintended side effects (a +/- lines changed under 200 is generally advised)
```

Comment:
```suggestion
- Keep PRs as small and focused as possible - this makes them easier for others to review and has a lower risk of introducing unintended side effects (a +/- for lines of source code changed under 200 is generally advised)
```

To clarify, docs PRs or the addition of test cases can often lead to lots of lines changed while still being quite readable/reviewable